gravity-forms/gw-create-coupon.php

```diff
@@ -238,6 +239,22 @@ function is_applicable_form( $form ) {
 		return (int) $form_id === (int) $this->_args['form_id'];
 	}
 
+	public function should_abort( $entry ) {
+		if ( empty( $this->_args['required_fields'] ) ) {
+			return false;
+		}
+
+		foreach ( $this->_args['required_fields'] as $field_id ) {
+			$value = rgar( $entry, (string) $field_id );
+
+			if ( empty( $value ) ) {
```

Comment:
`rgblank` may be a better option here, otherwise entering `0` will not satisfy the field requirement.

Reply:
@claygriffiths Fixed. Thanks!